test/androidTest/java/org/thoughtcrime/securesms/EspressoHelper.java

```diff
@@ -0,0 +1,250 @@
+package org.thoughtcrime.securesms;
+
```

Comment:
your imports are not corresponding to the coding style

Reply:
import android
import 3ed part : com, org
import java javax

check where  the junit goes , I believe it goes with 3rd part libs

Reply:
that's exactly how these are added, but the static ones are after. 

I didn't add any of these manually, so it's android studio's placement